src/Directory.Build.props

```diff
@@ -220,8 +220,9 @@
     <TestsNetCoreFramework>net6.0</TestsNetCoreFramework>
     <XunitVersion>2.4.1</XunitVersion>
     <TestSdkVersion>17.2.0</TestSdkVersion>
+	<XunitRunneVisualstudio>2.4.5</XunitRunneVisualstudio>
     <AkkaVersion>1.4.39</AkkaVersion>
-    <MicrosoftExtensionsVersion>[3.0.0,)</MicrosoftExtensionsVersion>
```

Comment:
Need to leave the original behavior here, as this has upstream breaking changes for some applications still running older versions of .NET Core / .NET 5.